packages/ruby.rb

```diff
@@ -28,7 +16,13 @@ class Ruby < Package
   # at run-time, system's gmp, openssl, readline and zlibpkg are possible to use
 
   def self.build
-    system "./configure #{CREW_OPTIONS} --enable-shared"
+    ENV['CFLAGS'] = '-ltinfo -fstack-protector-all'
+    system "./configure #{CREW_OPTIONS} \
+    --enable-shared \
+    --disable-fortify-source \
+    --disable-install-doc \
+    --disable-install-rdoc \
+    --disable-install-capi"
```

Comment:
Why do we need to disable?

Reply:
I was speeding up the builds and shrinking an essential application, which should speed up installs?

If people care about them, there's no other reason to take them out.

Reply:
How much space are we saving?  Is it worth it?

Reply:
1.5 Mb saved! So bupkis. I'll revert the disable-installs.